src/System.Runtime.Extensions/src/System/CannotUnloadAppDomainException.cs

```diff
@@ -0,0 +1,43 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+/*=============================================================================
+**
+** 
+**
+** Purpose: Exception class for failed attempt to unload an AppDomain.
+**
+**
+=============================================================================*/
+
+namespace System {
```

Comment:
Nit: Same comment about cleaning up the formatting to match corefx coding style.